geoportal/c2cgeoportal_geoportal/views/dynamic.py

```diff
@@ -100,7 +100,7 @@ def _interface(self, interface_config, interface_name, dynamic):
 
     @view_config(route_name="dynamic", renderer="fast_json")
     def dynamic(self):
-        interface_name = self.request.params.get("interface")
+        interface_name = self.request.get_organization_interface(self.request.params.get("interface"))
```

Comment:
It's not clear to me what an organisation interface is. How is the relationship between organisation and interface? Do we describe that somewhere?

Reply:
+1, Thanks, will be fixed :-)